utils/spacewalk-sync-setup

```diff
@@ -606,7 +614,11 @@ def describe_master_template(master_setup_filename):
 
 def ask(msg, password=False):
     msg += ": "
-    return password and getpass.getpass(msg) or raw_input(msg)
+    if six.PY2:
```

Comment:
We need to import `six` here, right?

Reply:
@meaksh fixed https://github.com/uyuni-project/uyuni/blob/1976508754ba3b7732ec5db4d3b9e24ff67879ea/utils/spacewalk-sync-setup#L31

Reply:
@meaksh fixed. https://github.com/uyuni-project/uyuni/blob/46419ad08c94af75f7b1cebf8321f2a8e2048cd1/utils/spacewalk-sync-setup#L31